### PR TITLE
Deployed Rebalancing Set Simulations

### PR DESCRIPTION
**Pull request by felix2feng**
Tests all scenarios

### CODE REVIEW

File: deployments/test/5_rebalancing.spec.ts [hunk 1]
```diff
@@ -314,13 +320,19 @@ describe('Deployment: Rebalancing', () => {
     });
 
     it('rebalancing manager has correct wBTC medianizer address', async () => {
-      const wBTCMedianizerAddress = dependencies.WBTC_MEDIANIZER[networkId];
+      const wBTCMedianizerAddress = dependencies.WBTC_MEDIANIZER[networkId] ?
```
Comment (bweick) [inline deployments/test/5_rebalancing.spec.ts]: You can just use `findDependency` here

File: deployments/test/5_rebalancing.spec.ts [hunk 2]
```diff
@@ -314,13 +320,19 @@ describe('Deployment: Rebalancing', () => {
...
+        dependencies.WBTC_MEDIANIZER[networkId] :
+        await getContractAddress(DEPENDENCY.WBTC_MEDIANIZER);
+
       const receivedBTCMedianizerAddress = await rebalancingManagerContract.methods.btcPriceFeed().call();
       expect(receivedBTCMedianizerAddress).toEqual(wBTCMedianizerAddress);
     });
 
     it('rebalancing manager has correct wETH medianizer address', async () => {
-      const wETHMedianizerAddress = dependencies.WETH_MEDIANIZER[networkId];
+      const wETHMedianizerAddress = dependencies.WETH_MEDIANIZER[networkId] ?
```
Comment (bweick) [inline deployments/test/5_rebalancing.spec.ts]: You can just use `findDependency` here

File: deployments/test/5_rebalancing.spec.ts [hunk 3]
```diff
@@ -549,7 +561,10 @@ describe('Deployment: Rebalancing', () => {
...
```
Comment (bweick) [inline deployments/test/5_rebalancing.spec.ts]: You can just use `findDependency` here

File: deployments/test/5_rebalancing.spec.ts [hunk 4]
```diff
@@ -78,13 +78,19 @@ describe('Deployment: Rebalancing', () => {
...
+        dependencies.WBTC_MEDIANIZER[networkId] :
+        await getContractAddress(DEPENDENCY.WBTC_MEDIANIZER);
+
       const receivedBTCMedianizerAddress = await rebalancingManagerContract.methods.btcPriceFeed().call();
       expect(receivedBTCMedianizerAddress).toEqual(wBTCMedianizerAddress);
     });
 
     it('rebalancing manager has correct wETH medianizer address', async () => {
-      const wETHMedianizerAddress = dependencies.WETH_MEDIANIZER[networkId];
+      const wETHMedianizerAddress = dependencies.WETH_MEDIANIZER[networkId] ?
```
[the remark this answers is not in the document]
Comment (asoong) [reply deployments/test/5_rebalancing.spec.ts]: +1 all of these

**Pull request merged**